src/shared/components/mutationTable/column/GenericNumericColumnFormatter.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Mutation } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+import generalStyles from 'shared/components/mutationTable/column/styles.module.scss';
+
+/**
+ * @author Pim van Nierop
+ */
+export default class GenericNumericColumnFormatter {
```

Comment:
same thing - generic name but the implementation concerns namespaces

Reply:
Updated. See comment above.